azkaban-common/src/main/java/azkaban/executor/ExecutionControllerUtils.java

```diff
@@ -255,6 +257,25 @@ public static void alertUserOnFirstError(final ExecutableFlow flow,
     }
   }
 
+  /**
+   * Alert the user when job property is overridden in a project
+   * @param project
+   * @param flow
+   * @param eventData
+   */
+  public static void alertUserOnJobPropertyOverridden(final Project project, final Flow flow,
+      final Map<String, Object> eventData, final AlerterHolder alerterHolder) {
+    if (!flow.getOverrideEmails().isEmpty()) {
+      logger.info("Alert on job property overridden event in project: " + project.getName());
+      final Alerter mailAlter = alerterHolder.get("email");
```

Comment:
Typo: `mailAlter` - I believe we wanted mailAlerter.

Reply:
thanks for pointing it out.